source/extensions/filters/http/kill_request/kill_request_filter.cc

```diff
@@ -61,8 +60,7 @@ Http::FilterHeadersStatus KillRequestFilter::decodeHeaders(Http::RequestHeaderMa
 
     if (per_route_kill_settings) {
       is_correct_direction = per_route_kill_settings->getDirection() == KillRequest::REQUEST;
-      envoy::type::v3::FractionalPercent probability = per_route_kill_settings->getProbability();
-      kill_request_.set_allocated_probability(&probability);
+      kill_request_.mutable_probability()->CopyFrom(per_route_kill_settings->getProbability());
```

Comment:
Nit: worth adding a comment for the change, since it was a bug that was missed.

Reply:
done.